components/Feature/TopicExplorer/index.js

```diff
@@ -26,10 +39,10 @@ const TopicExplorer = (props) => {
         <input
           type="text"
           className="govuk-input govuk-input--width-20"
-          placeholder="e.g. food, mental health, lockdown"
           value={searchTerm}
           onChange={onSearchChange}
         />
+        <div id='example-search'>Try a search for <span data-search-term='food' data-testid='food' onClick={populateInput} className='inlineLink'>food</span>, <span className='inlineLink' data-search-term='mental health' onClick={populateInput} >mental health</span>, or <span className='inlineLink' data-search-term='debt'onClick={populateInput} >debt</span></div>
```

Comment:
I'm not sure how clickable spans will work with screenreaders and other accessibility tools.

I think from a roles perspective, these are buttons - adding [the appropriate ARIA role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role) to these elements may be sufficient?

Reply:
Ive now changed this so anchor tags are being used